test/integration/merkle-lockup/factory/create-merkle-lockup-lt/createMerkleLockupLT.tree

```diff
@@ -0,0 +1,15 @@
+createMerkleLockupLL.t.sol
+├── when the tranches percentages sum does not equal 100%
+│   ├── when the tranches percentages sum is less than 100%
```

Comment:
can we not just say

- when the tranches percentages sum does not equal 100%
    - it should revert
- when the tranches percentages sum equals 100%
.......

Reply:
we could, be it's better to handle to cover all cases: when less and greater